code/style.css

```diff
@@ -1,4 +1,259 @@
+  background: url(img/tiger.png) no-repeat;
+  box-shadow: 10px 10px 5px #ccc;
+  -moz-box-shadow: 10px 10px 5px #ccc;
+  -webkit-box-shadow: 10px 10px 5px #ccc;
+  -khtml-box-shadow: 10px 10px 5px #ccc;
+  opacity: 1;
+  transition: .5 ease;
+  backface-visibility: hidden;
+
+}
+
+.cred {
+  font-size: 15px;
+}
+
+.grid-pic img {
+  animation-duration: 6s;
+  animation-iteration-count: infinite;
+  animation-name: logobounce;
+  animation-timing-function: cubic-bezier(0.280, 0.840, 0.420, 1);
+  margin: 10px;
+}
+
+@keyframes logobounce {
+  0%   { transform: scale(1,1)      translateY(0); }
+  10%  { transform: scale(1.1,.9)   translateY(0); }
+  30%  { transform: scale(.9,1.1)   translateY(-100px); }
+  50%  { transform: scale(1.05,.95) translateY(0); }
+  57%  { transform: scale(1,1)      translateY(-7px); }
+  64%  { transform: scale(1,1)      translateY(0); }
+  100% { transform: scale(1,1)      translateY(0); }
+}
+
+h1 {
+  font-size: 25px;
+  font-family: 'Averia Serif Libre';
+  font-weight: lighter;
+  letter-spacing: 3px;
+  width: 100%;
+  padding: 18px 0px 0px 10px;
+  animation-name: example;
+  animation-duration: 5s;
+  animation-iteration-count: infinite;
+}
+
+ @keyframes example {
+   0% {color: #B36A5E;}
+   25% {color: #C97C5D;}
+   50% {color: #C89F9C;}
+   75% {color: #EED7C5;}
+   100% {color: #EEE2DF;}
+ }
+
+h2 {
+  font-size: 30px;
+  padding: 2px;
+  height: 30px;
+  cursor: pointer;
+}
+
+h3 {
+  font-size: 25px;
+  cursor: pointer;
+}
+
+ h2, h3, p {
+ font-family: 'Source Sans Pro', sans-serif;
+ }
+
+ h2:hover {
+   color: #1F7980;
+ }
+
+ h3:hover {
+  color: #1F7980;
+}
+
+ img {
+  width: 100%;
+}
+
+footer {
+  padding-top: 20px;
+  text-align: center;
+  background-color: #EED7C5;
+  width: 100%;
+  grid-column: span 4;
+  line-height: 7px;
+}
+
+.signature {
+  font-family: 'Nothing You Could Do';
+
+}
+
+ /* tablets*/
+ @media (min-width: 600px) and (max-width: 991px) {
+   
+  .full__container {
+    width: 100%;
+    font-size: 40px;
+  }
+
+  h1 {
+    font-size: 37px;
+    padding-left: 10px;
+  }
+
+  h2 {
+    font-size: 30px;
+    height: 30px;
+  }
+
+  .body-section-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    padding: 20px;
+    grid-gap: 15px;
+    background: white;
+    height: 400px;
+  }
+  
+  .smallnews-grid {
+    grid-column: span 2;
+    display: grid;
+    flex-direction: column;
+    justify-content: center;
+  
+  }
+
+}
+
+
+/*desktop*/
+ @media (min-width: 992px) {
+
+   .full-container {
+     width: 100%;
+   }
+   
+   h1 {
+    font-size: 65px;
+  }
+
+  h2 {
+    font-size: 40px;
+  }
+
+  p {
+    font-size: 20px;
+  }
+
+  .body-section-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    padding: 20px;
+    grid-gap: 20px;
+    background: white;
+    height: 400px;
+  }
+  
+  .smallnews-grid {
+    grid-column: span 1;
+    display: grid;
+    flex-direction: column;
```

Comment:
I guess you had played around with different options and forgot to delete this line (254: flex-direction:column) thus its from flex boxes and also if I disable it in dev tools nothing is changing . I suppose the line 252 carries out its function and creates the columns as required. So I suppose 254 can be just deleted.

Reply:
Your right, I guess I mixed up flexbox and grid when I tried them both. Cleaned it up!